corehq/apps/app_manager/fixtures/mobile_ucr.py

```diff
@@ -18,6 +18,11 @@
 from corehq.apps.app_manager.dbaccessors import get_apps_in_domain
 
 
+def _should_sync(last_sync_log):
+    sync_interval = 4 * 60 * 60
```

Comment:
might be difficult but would also be nice if it respected the `overwrite_cache` param